modules/gltf/src/lib/extensions/deprecated/EXT_feature_metadata.ts

```diff
@@ -138,26 +358,29 @@ function findFeatureTableByName(
  * @param stringsCount
  */
 function getStringAttributes(
-  data: Uint8Array,
+  data: Uint8Array | string[],
```

Comment:
`string[]` is it the case for some data source? I've checked in the previous version of this file and it seems it is the output type

Reply:
Resolved. You are right. It was my fault.